standalone/src/init.ts

```diff
@@ -105,21 +96,12 @@ function connectCall(channelID: string, callTitle: string, wsURL: string, iceCon
         window.callsClient = new CallsClient({
             wsURL,
             iceServers: iceConfigs,
+            authToken: getToken(),
```

Comment:
This is not needed for the widget use case but it will be for the recording view and potentially more use cases in the future (e.g. external calls).